docs/kb/supporting-your-customers/restoring-from-a-snapshot.md

```diff
@@ -8,11 +8,7 @@ categories = [ "Knowledgebase", "Supporting Your Customers" ]
 
 **The snapshots functionality in Replicated is for disaster recovery purposes.**
 
-If you have enabled snapshots for your app they will by default run every 24 hours, 
-optionally your users can run a snapshot from the on-prem console at any time. The default 
-location for saving a snapshot on a replicated enabled host is: `/var/lib/replicated/snapshots`, 
-the last 3 snapshots will be located in that directory.
-*Note: We highly recommend you copy this folder to a additional location on a different physical 
+Automatic snapshots by default are disabled, but If your customer has enabled snapshots they will by default run every 24 hours, see the [Snapshots documentation](/packaging-an-application/snapshots/) for information on enabling snapshots. Optionally your users can run a snapshot from the on-prem console at any time. The default location for saving a snapshot on a replicated enabled host is: `/var/lib/replicated/snapshots`, the last 3 snapshots will be located in that directory. *Note: We highly recommend you copy this folder to a additional location on a different physical 
```

Comment:
That should be lower case "If"....  And I still think this should just link to /packaging-an-application/snapshots/ and not explain how to configure snapshots since it's explained in detail on that page.